packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.styling.ts

```diff
@@ -0,0 +1,24 @@
+import { radioGroupName, RadioGroupTokens, RadioGroupSlotProps, RadioGroupProps } from './RadioGroup.types';
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { defaultRadioGroupTokens } from './RadioGroupTokens';
+
+export const stylingSettings: UseStylingOptions<RadioGroupProps, RadioGroupSlotProps, RadioGroupTokens> = {
+  tokens: [defaultRadioGroupTokens, radioGroupName],
+  slotProps: {
+    root: buildProps(() => ({
+      accessible: true,
+      accessibilityRole: 'radio',
+      style: {
+        display: 'flex',
+        alignItems: 'flex-start',
+        flexDirection: 'column',
+      },
+    })),
+    label: buildProps(() => ({
+      style: {
+        fontSize: 16,
```

Comment:
Instead of hard coding numeric values for font styling, we may want to start using alias tokens. That way they will automatically set appropriate values for each platform if they do diverge.

Reply:
Makes sense! Revised with alias and global tokens now. Is there a way to define it using only VariantValue (not keyof Variants) to avoid having to define fontFamily, fontSize, and fontWeight separately?